include/seqan3/alignment/all.hpp

```diff
@@ -56,45 +56,47 @@
  *
  * \include test/snippet/alignment/pairwise/align_pairwise_range.cpp
  *
- * In addition to the type requirements above the alignment interface requires std::ranges::random_access_range and
- * std::ranges::sized_range in order to work correctly.
+ * \attention In addition to the type requirements above the alignment interface requires that the passed sequences
+ *            model std::ranges::random_access_range and std::ranges::sized_range in order to work correctly, e.g. a
+ *            std::vector.
  *
  * ## Configuring pairwise alignments
  *
  * In SeqAn the alignment algorithm can be configured in many different ways. The core of this configuration are the
  * different configuration elements that select specific features of the algorithm. To allow a maximal flexibility
- * the configuration is separated from the alignment interface. This means that before the alignment algorithm
- * is invoked, the algorithm must be configured. The respective alignment configurations are defined in their own
+ * the configuration is separated from the alignment interface. This means the algorithm must be configured before it is
+ * invoked. The respective alignment configurations are defined in their own
  * namespace called seqan3::align_cfg. This namespace is used to disambiguate configurations for the
  * alignment algorithm with configurations from other algorithms in SeqAn.
- * To compute a pairwise alignment at least two configuration elements must be provided, namely the
- * the alignment method and the seqan3::align_cfg::scoring_scheme.
+ * To compute a pairwise alignment at least two configuration elements must be provided: The alignment method and the
+ * scoring scheme. Thus, a valid alignment configuration must specify what kind of alignment is to be computed, since
+ * this strongly depends on the used context and a default wouldn't make much sense here. For similar reasons the
```

Comment:
Still open.

(see previous discussion https://github.com/seqan/seqan3/pull/2191#discussion_r500959158)

Reply:
It is a rather vague comment. There is no clear message what should be changed. I tried to interpret it as the sentence is to complicated so I split it up. I hope that suffices?